applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -461,12 +461,8 @@ static int change_active_brdcast_src(void)
 {
 	int ret;
 
-	LOG_INF("Change broadcast source");
-
-	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
-	ERR_CHK(ret);
-
-	ret = bis_headset_cleanup(false);
+	/* If the stream has been paused do not stop the sink again */
+	ret = bis_headset_cleanup(!playing_state);
```

Comment:
I have not reviewed this PR completely, but it looks like we must rename "bool from_sync_lost_cb"

Reply:
Yes, I thought about that, but didn't want to change it in this PR. I think there is more than that in the sync list that would need changing and would be a PR in itself!